core/src/main/scala/com/databricks/labs/remorph/transpilers/Transpiler.scala

```diff
@@ -1,6 +1,18 @@
 package com.databricks.labs.remorph.transpilers
 
+import com.databricks.labs.remorph.parsers.{intermediate => ir}
+
 trait Transpiler {
 
-  def transpile(input: String): String
```

Comment:
keep the public API of this trait only to this one single method. no other methods allowed

Reply:
Good catch, I should have made these methods protected